Makefile

```diff
@@ -18,16 +20,27 @@ theme:
 npm: theme
 	npm ci
 
+
+#hugo-check: @ checks system version of hugo 
+hugo-check:
+ifneq ($(hugo_local),$(hugo_prod))
+	@echo hugo $(hugo_local) validated
+else
+	@echo $@ failure: Your hugo version \($(hugo_local)\) does not match production \($(hugo_prod)\) 
+	@echo run \'make hugo-update\' or \'brew upgrade hugo\'
```

Comment:
Haven't written the rule for hugo-update yet, but also may not need if we just direct them to upgrade via brew manually. Also had an idea to write a rule for automating the prerequisite stuff for new (and maybe even existing) contributors.

Reply:
This looks great! I think we should avoid trying to auto install or update hugo for now. It could get pretty hairy as we'd have to check if they're on a Mac, what version they have to see if we have to install or upgrade, etc. The logic is doable but complex so I think we just change the message to say use brew upgrade or brew install or wget etc. for now.